packages/react-app-bridge-universal-provider/src/tests/AppBridgeUniversalProvider.test.tsx

```diff
@@ -8,6 +8,21 @@ import {HtmlManager, HtmlContext} from '@shopify/react-html';
 import {AppBridgeUniversalProvider} from '../AppBridgeUniversalProvider';
 
 describe('<AppBridgeUniversalProvider />', () => {
+  const assign = window.location.assign;
+
+  beforeAll(() => {
```

Comment:
Could we use [`@shopify/jest-dom-mocks`'](https://github.com/Shopify/quilt/tree/main/packages/jest-dom-mocks#standard-mocks) location mock? it looks like that  handles stubbing/restoring [assign](https://github.com/Shopify/quilt/blob/main/packages/jest-dom-mocks/src/location.ts#L16)

Reply:
Good catch. Fixed.